### PR TITLE
Update GetSessionToken error logging

### PR DESCRIPTION
**Pull request by toddtreece**

### CODE REVIEW

File: pkg/plugin/twinmaker/client.go [hunk 1]
```diff
@@ -534,31 +534,26 @@ func (c *twinMakerClient) GetSessionToken(ctx context.Context, duration time.Dur
 }
 
 func (c *twinMakerClient) GetWriteSessionToken(ctx context.Context, duration time.Duration, workspaceId string) (*sts.Credentials, error) {
-	_, err := c.writerService()
-	if err != nil {
-		return nil, err
+	if c.tokenRoleWriter != "" {
```
Comment (toddtreece) [inline pkg/plugin/twinmaker/client.go:537]: failing early if the writer role is not configured

File: pkg/plugin/twinmaker/client.go [hunk 2]
```diff
@@ -534,31 +534,26 @@ func (c *twinMakerClient) GetSessionToken(ctx context.Context, duration time.Dur
 }
 
 func (c *twinMakerClient) GetWriteSessionToken(ctx context.Context, duration time.Duration, workspaceId string) (*sts.Credentials, error) {
-	_, err := c.writerService()
```
Comment (toddtreece) [inline pkg/plugin/twinmaker/client.go:537]: is this `writerService` call needed? it seems like `GetWriteSessionToken` handles assuming the role

File: pkg/plugin/twinmaker/handler.go [hunk 3]
```diff
@@ -623,25 +623,18 @@ func (s *twinMakerHandler) GetWriteSessionToken(ctx context.Context, duration ti
...
-		case sts.ErrCodeRegionDisabledException:
-			fmt.Println(sts.ErrCodeRegionDisabledException, aerr.Error())
-		default:
-			fmt.Println(aerr.Error())
-		}
-	} else {
-		// Print the error, cast err to awserr.Error to get the Code and
-		// Message from an error.
-		fmt.Println(err.Error())
+		log.DefaultLogger.Error("error getting session token", "code", aerr.Code(), "error", aerr)
```
Comment (toddtreece) [inline pkg/plugin/twinmaker/handler.go:633]: @haweston i updated this to use the grafana plugin SDK logger. it seems like having the code might be useful info in all cases where the info is available, so i added a `code` field

File: pkg/plugin/twinmaker/client.go (hunk 2 above)
Comment (haweston) [reply pkg/plugin/twinmaker/client.go:537]: I don't think. This was just mirroring the other getsession function really.

File: pkg/plugin/twinmaker/handler.go (hunk 3 above)
Comment (haweston) [reply pkg/plugin/twinmaker/handler.go:633]: Okay, yeah that's definitely better.

**Review by haweston — APPROVED** [review]

**Pull request merged**